src/Controller/HomeController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\Routing\Annotation\Route;
+use Facebook\Facebook;
+use Facebook\Exceptions\FacebookResponseException;
+use Facebook\Exceptions\FacebookSDKException;
+use Symfony\Component\HttpFoundation\Response;
+
+class HomeController extends Controller
+{
+    /**
+     * @Route("/")
+     */
+    public function index()
+    {
+        $fb = new Facebook([
+            'app_id' => '2100826210141247', // Replace {app-id} with your app id
+            'app_secret' => 'a649d831412f50ba11a6fdb774318498',
```

Comment:
why you don't use `parameters.yml` for credentials?

Reply:
There is no parameters.yml in Symfony4. Even more, you shold regenerate token because any way it is in git history so it's a security issue